tests/libnet/job/job.go

```diff
@@ -1,36 +1,75 @@
-package tests
+package job
```

Comment:
Please add the license header.

Reply:
Maybe will be good to add presubmit that checks new files have a header

Reply:
Please see https://github.com/kubevirt/kubevirt/pull/10119